.github/workflows/test_ci.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will install Python dependencies and run tests with a single version of Python for Penn repository
+
+name: Testing Penn
+
+on:
+  push:
+    branches: [ "master" ]
```

Comment:
I'd remove the rows restricting the CI to the master branch so other branches would be tested as well.